lib/execution-profile.js

```diff
@@ -202,18 +173,11 @@ class ProfileManager {
     }
 
     /**
-     * Gets the cached default graph options for a given profile. If it doesn't exist, it creates new options using the
-     * handler and inserts it into the cache
-     * @param {ExecutionProfile} profile
-     * @param {Function} createHandler
+     * @deprecated Not supported by the driver. Usage will throw an error.
+     * (But is it used anywhere?)
      */
-    getOrCreateGraphOptions(profile, createHandler) {
-        let graphOptions = this._graphOptionsCache[profile.name];
-        if (!graphOptions) {
-            graphOptions = this._graphOptionsCache[profile.name] =
-                createHandler();
-        }
-        return graphOptions;
+    getOrCreateGraphOptions() {
+        throwNotSupported("ProfileManager.getOrCreateGraphOptions");
```

Comment:
Regarding the question in the comment: let's not leave such questions in comments in merged code. We should answer them before merging.